HomeWork06/task1.py

```diff
@@ -0,0 +1,8 @@
+def exponentiation(number, power_to_raise):
+    """
+    This is a function that returns number raised to the power of the second argument
+    :param number: int
+    :param power_to_raise: int
+    :return: int
+    """
+    return number**power_to_raise
```

Comment:
nit: add spaces before and after operations
```suggestion
    return number ** power_to_raise
```